packages/common/src/store/reachability/reducer.ts

```diff
@@ -6,7 +6,7 @@ import { ReachabilityState } from './types'
 type ReachabilityActions = ActionType<typeof actions>
 
 const initialState = {
-  networkReachable: true
+  networkReachable: false
```

Comment:
This is a shift in the initial assumption of connectivity. It helps the sagas block properly when they should.

TODO: test this in web and desktop to make sure it doesn't break initial load

Reply:
seems to be working on desktop and web